lib/benchee/conversion.ex

```diff
@@ -51,6 +58,11 @@ defmodule Benchee.Conversion do
       |> Enum.flat_map(fn scenario -> Map.to_list(scenario.run_time_data.statistics) end)
       |> Enum.group_by(fn {stat_name, _} -> stat_name end, fn {_, value} -> value end)
 
+    reductions_measurements =
+      scenarios
+      |> Enum.flat_map(fn scenario -> Map.to_list(scenario.reductions_data.statistics) end)
+      |> Enum.group_by(fn {stat_name, _} -> stat_name end, fn {_, value} -> value end)
+
     memory_measurements =
       scenarios
       |> Enum.flat_map(fn
```

Comment:
the line below here matches against `statistics: nil` which is no longer happening/valid with the `defstruct` changes from above

Reply:
I think we actually don't need this anymore. This is legacy stuff from before we unified the data structure for each collection type.